pr_check.sh

```diff
@@ -1,12 +1,25 @@
 #!/bin/bash
 
-set -exv
+#Adapted from https://github.com/RedHatInsights/insights-ingress-go/blob/master/pr_check.sh
 
-# Make directory for artifacts
-mkdir -p artifacts
+# --------------------------------------------
+# Options that must be configured by app owner
+# --------------------------------------------
+APP_NAME="drift"  # name of app-sre "application" folder this component lives in
+COMPONENT_NAME="drift"  # name of app-sre "resourceTemplate" in deploy.yaml for this component
+IMAGE="quay.io/cloudservices/drift-backend"
 
-cat << EOF > artifacts/junit-dummy.xml
-<testsuite tests="1">
-    <testcase classname="dummy" name="dummytest"/>
-</testsuite>
-EOF
+IQE_PLUGINS="drift"
+IQE_MARKER_EXPRESSION="smoke" # Need to check this
+IQE_FILTER_EXPRESSION=""
+IQE_CJI_TIMEOUT="30m"
+
+
+# Install bonfire repo/initialize
+CICD_URL=https://raw.githubusercontent.com/RedHatInsights/bonfire/master/cicd
+curl -s $CICD_URL/bootstrap.sh > .cicd_bootstrap.sh && source .cicd_bootstrap.sh
+
+source $CICD_ROOT/build.sh
+docker run $IMAGE:$IMAGE_TAG /bin/bash -c "pipenv install --dev && ./run_unit_tests.sh"
```

Comment:
Might be good if you can extract a junit.xml result from the unit tests here. If there is any file named `junit-*.xml` in the `artifacts` dir of $WORKSPACE, the Jenkins job will parse the test results.